src/main/steps/common/form/form.njk

```diff
@@ -52,7 +52,7 @@
             disabled: true if form.submit.disabled else false,
             preventDoubleClick: true,
             attributes: {
-              id: 'main-form-submit'
+              id: 'main-form-save-and-continue'
```

Comment:
Why is the id changed? Is the id used in code any where? Does it impact any of the functionality? Have you tested the changes?

Reply:
Its duplicated in same file so changed it
Its not used anywhere across the application